packages/mini-apps-ui-kit-react/src/components/AlertDialog/AlertDialog.tsx

```diff
@@ -54,22 +55,32 @@ AlertDialogClose.displayName = "AlertDialogClose";
 const AlertDialogContent = React.forwardRef<
   React.ElementRef<typeof DrawerPrimitive.Content>,
   AlertDialogContentProps
->(({ className, children, ...props }, ref) => (
-  <AlertDialogPortal>
-    <DrawerPrimitive.Overlay
-      ref={ref}
-      className={cn("fixed inset-0 z-50 bg-gray-900/40", className)}
-      {...props}
-    />
-    <DrawerPrimitive.Content
-      ref={ref}
-      className={cn("fixed inset-x-0 bottom-0 z-50 mt-24 h-auto", className)}
-      {...props}
-    >
-      <div className="flex flex-col rounded-[1.75rem] bg-gray-0 m-3 p-8">{children}</div>
-    </DrawerPrimitive.Content>
-  </AlertDialogPortal>
-));
+>(({ className, children, ...props }, ref) => {
+  const safe = useSafeAreaInsets();
+  return (
+    <AlertDialogPortal>
+      <DrawerPrimitive.Overlay
+        ref={ref}
+        className={cn("fixed inset-0 z-50 bg-gray-900/40", className)}
+        {...props}
+      />
+      <DrawerPrimitive.Content
+        ref={ref}
+        className={cn("fixed inset-x-0 bottom-0 z-50 mt-24 h-auto", className)}
+        {...props}
+      >
+        <div
+          className="flex flex-col rounded-[1.75rem] bg-gray-0 m-3 p-8"
+          style={{
+            marginBottom: Math.max(safe.bottom, 12),
+          }}
```

Comment:
```suggestion
          className="flex flex-col rounded-[1.75rem] bg-gray-0 m-3 p-8 mb-[var(--safeBottomMargin)]"
          style={{
            "--safeBottomMargin": `${Math.max(safe.bottom, 12)}px`
          } as CSSProperties}
```

Reply:
Also need to add an import of CSS properties.

Reply:
@danielppolo Also why Math.max? If you need a fallback then use `safe.botton ?? 12` or the app sends 0 as default value?

Reply:
I'm using `.max` to have always a minimum of 12. Not only to handle 0 but 4px for example.

Reply:
On the variable, while I do like the approach I don't think is needed in this case. We don't allow to override the classes in this component and it might be unnecessary to add a CSS variable for one use.